packages/cinterop/src/darwinCommon/kotlin/io/realm/interop/RealmInterop.kt

```diff
@@ -75,8 +78,16 @@ private fun throwOnError(pointer: CPointer<out CPointed>?): CPointer<out CPointe
 }
 
 // FIXME API-INTERNAL Consider making NativePointer/CPointerWrapper generic to enforce typing
-class CPointerWrapper(ptr: CPointer<out CPointed>?) : NativePointer {
+
+class CPointerWrapper(ptr: CPointer<out CPointed>?, managed: Boolean = true) : NativePointer {
```

Comment:
I guess we should remember to make the notification tokens unmanaged?

Reply:
Good catch 👍